models/ccmm.yaml

```diff
@@ -18,6 +18,47 @@ $defs:
       - id
       - title
       - props.iri
+  vocabulary-languages:
+    use: "./ccmm.yaml#/$defs/vocabulary"
```

Comment:
Why not just `#/$defs/vocabulary`?

Reply:
need to have specialised handling for iris in marshmallow different for each vocab type